README.md

```diff
@@ -1 +1,86 @@
 # Papers of the Week
+
+
+
+## PoW Series
+
+The IE Robotics Club organizes the AI Paper of the Week, a **discussion on a trending topic related to Robotics or AI**. During the event, we will be reviewing and analyzing a paper on one of these topics and **discussing the most intriguing aspects** of it in depth. Any doubts that may arise during the reading will be addressed.
+
+📅 The paper to be discussed will be **chosen one week in advance** by all attendees of the upcoming event and will be published for **individual reading beforehand**.
+
+🙋🏻‍♂️ You do **not need to be an expert** in the field or have an in-depth understanding of every concept presented in the paper. If you have a **passion for AI or a general interest in robotics** and wish to learn more, you are welcome to join the discussion as an active participant or simply listen in without any obligation.
+
+✅ To ensure that everyone has access to the information presented during the event, we will **create slides and share them** via Notion. If you are unable to attend, you may access the slides from the **previous session and this week's paper** by accessing here: [Papers](https://www.notion.so/AI-PoW-Series-77071497fa224d2a897e77ec0f4b172a)
+
+## Past Papers
+
+
+ 
+| Topic | Paper | Code | Visuals |
+| :---: | :---: | :---: | :---: |
+| GANS  | Generative Adversial Networks | [Code](./gans/) | <img src="./gans/evolution.gif" width="100" height="100"> |
+
+
+## How to make this code run
+
+#### 1. Create a virtual environment and activate it
+
+```bash
+make install TOPIC=gans
+```
+
+
+#### 2. Activate the virtual environment
+
+```bash
+cd <topic>
+source .venv/bin/activate
+```
+
+
```

Comment:
This should be the README for GANs, not for the PoWs repo

Reply:
If you want to keep it with the <topic_folder_name> variable. Perhaps reformat it to something like:

```
## How to create an environment with the topic of the week

#### 1. Create a virtual environment and activate it

```
make install TOPIC=<topic_folder_name>
```

Also you don't need to activate it since this is already present in the Makefile